src/components/ammo-constraint.js

```diff
@@ -69,7 +69,7 @@ module.exports = AFRAME.registerComponent("ammo-constraint", {
 
     const bodyTransform = body
       .getCenterOfMassTransform()
-      .invert()
+      .inverse()
```

Comment:
Yeah, that's super sneaky. Was indeed confusing. But inverse() does seem to be what's actually available here: https://github.com/kripken/ammo.js/blob/99d0ec0b1e26d7ccc13e013caba8e8a5c98d953b/bullet/src/BulletDynamics/ConstraintSolver/btGeneric6DofSpringConstraint.cpp#L178-L179

You might add a comment here about this, because this could easily be confused by someone else in the future.